### ISSUE TITLE
janus_client.go:472: Unable to deliver message

### ISSUE THREAD

**Issue by SystemKeeper** [strukturag/nextcloud-spreed-signaling#58]
I'm seeing the following messages in the logs:

```
2020-10-28 19:28:52.180728 I | mcu_janus.go:992: Subscriber 7034386731986703 received hangup (DTLS alert), closing
2020-10-28 19:28:52.238023 I | mcu_janus.go:701: Publisher 2940641805820439 received hangup (DTLS alert), closing
2020-10-28 19:28:52.238486 I | mcu_janus.go:688: Publisher 2940641805820439: associated room has been destroyed, closing
2020-10-28 19:28:52.238748 I | mcu_janus.go:760: Room 8404584723790815 destroyed
2020-10-28 19:28:52.239283 I | janus_client.go:472: Unable to deliver message {
   "janus": "detached",
   "session_id": 2596656192615391,
   "sender": 2940641805820439
}. Handle 2940641805820439 gone?
```

Not sure if this is a bug, because it's working correctly as far as I can tell...

**Comment by fancycode** [comment]
This is the server still receiving a notification for a janus room that already has been closed shortly before (`Publisher 2940641805820439: associated room has been destroyed, closing`). The receiver for this room already got removed, so it can't be delivered. Not a problem.

**Comment by SystemKeeper** [comment]
Thanks for the explanation!
